image_provisioner/playbooks/roles/collectd-install/files/script.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
```

Comment:
Instead of a configmap, we can build the collectd image with the collecd.conf.j2 template copied in to the container. I also updated the PR with a collecd-daemonset.yaml which can be used to launch collectd pods on all the nodes.